reference-architecture/osp-cli/ch4.8.4_enable_docker_nodes.sh

```diff
@@ -0,0 +1,5 @@
+for H in $NODES
+do
+  ssh $H sudo systemctl enable docker
```

Comment:
You should be able to remove this as the installer should take care of starting and enabling